test/units/modules/cloud/kubevirt/test_kubevirt_vm.py

```diff
@@ -0,0 +1,111 @@
+import json
+import pytest
+import sys
+
+from openshift.dynamic import Resource
+from openshift.helper.exceptions import KubernetesException
+
+from units.compat.mock import patch, MagicMock
+
+from ansible.module_utils.k8s.common import K8sAnsibleMixin
+from ansible.module_utils.k8s.raw import KubernetesRawModule
+
+from kubevirttestutils import set_module_args, AnsibleFailJson, AnsibleExitJson, exit_json, fail_json, RESOURCE_DEFAULT_ARGS
+
+from ansible.modules.cloud.kubevirt import kubevirt_vm as mymodule
+
+KIND = 'VirtulMachine'
+
+
+class TestKubeVirtVmModule(object):
```

Comment:
New unit tests should be pytest style (stand-alone functions) instead of classes.